include/jemalloc/internal/hpa.h

```diff
@@ -67,6 +69,12 @@ struct hpa_shard_s {
 	/* The arena ind we're associated with. */
 	unsigned ind;
 	emap_t *emap;
+
+	/*
+	 * The number of times we've purged a hugepage.  Guarded by the grow
```

Comment:
nit: mention that each eviction purges a single hugepage.

Reply:
Done. (Actually, this was also done in the purging stack, so it's doubly done now since I liked that phrasing better).